stan/math/prim/scal/prob/normal_cdf.hpp

```diff
@@ -45,9 +45,7 @@ namespace stan {
 
       T_partials_return cdf(1.0);
```

Comment:
This is a different order of return vs. all the other cases.  Should we fix one?  I don't think there's an efficiency concern either way, but I think having the return be a constant and the return defined afterwards is easier to understand (doesn't require you to go look back to find value of `cdf` being returned).